book/src/package-managers.md

```diff
@@ -101,7 +101,15 @@ Download the official Helix AppImage from the [latest releases](https://github.c
 chmod +x helix-*.AppImage # change permission for executable mode
 ./helix-*.AppImage # run helix
 ```
- 
+
+You can optionally [add the `.desktop` file](./building-from-source.md#configure-the-desktop-shortcut), but there's some pre-requisites:
+```sh
+mkdir -p "$HOME/.local/bin" && \
+mv helix-*.AppImage "$HOME/.local/bin/hx"
+```
+
+`hx` can be anywhere in your `$PATH`, but (by [convention](https://askubuntu.com/a/1408444)) `~/.local/bin` is preferred.
```

Comment:
```suggestion
and make sure `~/.local/bin` is in your `PATH`.
```

justifying the use of `~/.local/bin` seems unnecessary/offtopic to me, especially if we frame the directions above as an example